app/screens/ARView.js

```diff
@@ -1,10 +1,10 @@
 import React, { Component, View } from 'react-native';
-import AR from './AR.js';
-import Map from './Map.js';
+import AR from '../components/AR';
```

Comment:
@tiffeli We will definitely need three smart view components, due to the way `react-native-router-flux`/`Navigator` maps scenes to components. We might be able to combine the logic into a single file that exports multiple smart view components, or factor out some of the helper functions, but, as far as I understand it, there will need to be multiple smart view components.

Also, while similar, the views can actually have different data requirements, especially moving forward. For example, the list view doesn't need a users position, and the map view doesn't necessarily need it updated as regularly as AR view.

Filter mechanisms can/should operate on a shared set of visible data, and as of the last discussion would be implemented using actions on a cross-view drawer (aka SideMenu).